Assets/Scripts/EOSAchievementManager.cs

```diff
@@ -22,29 +22,31 @@
 
 //#define ENABLE_DEBUG_EOSACHIEVEMENTMANAGER
 
-using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.Networking;
-using System.Collections.Concurrent;
-using Epic.OnlineServices.Achievements;
-using Epic.OnlineServices.Stats;
-using System;
-using Epic.OnlineServices;
-using System.Diagnostics;
-
 namespace PlayEveryWare.EpicOnlineServices.Samples
 {
+    using System;
```

Comment:
stylistic non-criticism: I've always liked putting using statements under the namespace declaration, like this. Mint will bring this up as a stylistic thing for the team to consider doing.

Reply:
I concur.